Rakefile

```diff
@@ -83,15 +82,6 @@ namespace :dependencies do
     end
     CLOBBER << 'Pods'
   end
-
-  namespace :lint do
-    task :check do
-      if swiftlint_needs_install
-        dependency_failed('SwiftLint')
-        Rake::Task['dependencies:pod:install'].invoke
-      end
-    end
-  end
```

Comment:
We no longer need to check if the tool is installed because when we run it via `swift package plugin` (see below) it will be fetched if not available locally.